src/components/layers/PointLayer/use-symbol-image-loader.tsx

```diff
@@ -72,8 +72,11 @@ const useSymbolImageLoader = (
             }));
             throw err;
           }
-
-          map.addImage(m.name, image);
+          try {
```

Comment:
Could we potentially use https://docs.mapbox.com/mapbox-gl-js/api/map/#map#hasimage to check if the image is already added with this ID?

Still should have the try/catch though.